emulators/xhyve/Portfile

```diff
@@ -4,13 +4,13 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           xcode 1.0
 
-github.setup        machyve xhyve 1f46a3d0bbeb6c90883f302425844fcc3800a776
-version             20191002
+github.setup        machyve xhyve eab8ad838868205b872b93129e2ee91ca5328ea9
+version             20210922
 categories          emulators
 platforms           darwin
 supported_archs     x86_64
 license             BSD
-maintainers         {jeremyhu @jeremyhu}
+maintainers         {jeremyhu @jeremyhu} {@catap korins.ky:kirill}
```

Comment:
@catap I suggest adding "openmaintainer" too.

Reply:
@pmetzger let make things easy for everyone, yeah.